internal/credentials/kubernetes/database.go

```diff
@@ -232,7 +238,7 @@ func (k *database) getCredentialsSecret(
 		case credentials.TypeHelm:
 			secretURL = helm.NormalizeChartRepositoryURL(secretURL)
 		}
```

Comment:
I think the switch above needs a new case for image credentials similar to line +194

Reply:
Other than that, nailed it.